css/style.css

```diff
@@ -0,0 +1,230 @@
+/* Busra's Cake Company */
+
+/* grid-container */
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 120px auto 100px;
+  grid-gap: 10px;
+  background-color: ivory;
+}
+
+/* header */
+header {
+  background-color: rgb(252, 204, 213);
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  grid-template-rows: 50% 50%;
+  grid-column: 1 / -1;
+
+  padding: 10px;
+}
+
+header p {
+  grid-column: 5 / -1;
+  grid-row: 2 / 3;
+  text-align: right;
+  font-style: oblique;
+  font-weight: bold;
+}
+
+.logo {
+  grid-column: 1 / 3;
+  grid-row: 1 / 3;
+  width: 100%;
+  background-image: url("../img/Vanilla-Cupcakes-with-Vanilla-Swiss-Meringue-SQ.jpeg");
+  border-radius: 5%;
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+}
+
+.hamburger {
+  grid-column: 6 / -1;
+  grid-row: 1 / 2;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.hamburger img {
+  height: 40%;
+}
+
+/* nav on mobile */
+nav {
+  display: none;
+}
+
+/* main */
+main {
+  grid-column: 1 / -1;
+  background-color: ivory;
+}
+
+/* welcome */
+.welcome {
+  grid-column: 1 / -1;
+  margin-bottom: 1em;
+  background-color: ivory;
+}
+
+.welcome h1 {
+  font-size: xx-large;
+  font-style: oblique;
+}
+
+.welcome p {
+  font-style: initial;
+  text-align: center;
+}
+
+.welcome h2 {
+  font-size: large;
+  font-style: oblique;
+}
+
+.welcome-image {
+  background-color: ivory;
+  padding: 50px;
+}
+
+.welcome-image img {
+  width: 100%;
+  border-radius: 3px;
+}
+
+/* products */
+.products {
+  display: grid;
+  background-color: ivory;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 200px 200px;
+  grid-gap: 1rem;
+}
+
+.images {
+  background-size: 100%;
+  background-repeat: no-repeat;
+  width: 100%;
+  border-radius: 3px;
+}
+
+.product-image1 {
+  background-image: url("../img/pink-birthday-cake.jpeg");
+}
+
+.product-image2 {
+  background-image: url("../img/strawbery-cake.jpeg");
+}
+
+.product-image3 {
+  background-image: url("../img/white-cake.jpeg");
+}
+
+.product-image4 {
+  background-image: url("../img/wedding-cake.jpeg");
+}
+
+/* grid-area footer */
+footer {
+  background-color: ivory;
+  grid-column: 1 / -1;
+}
+
+/* footer */
+footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  padding-top: 25px;
+}
+
+.icon-container img {
+  width: 1rem;
+  height: 1rem;
+  padding: 0.28rem;
+}
+
+/* Media Queries */
+
+/* TABLET */
+@media screen and (min-width: 540px) {
+  .products {
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: 300px;
+    grid-column: 1 / -1;
+  }
+  .welcome {
+    text-align: center;
+  }
+}
+/* DESKTOP */
+
+@media screen and (min-width: 900px) {
```

Comment:
You are following mobile-first approach (mobile styles are defaults) - very good practice!